test/test_helper.rb

```diff
@@ -11,7 +11,7 @@ class ActiveSupport::TestCase
 
   # Add more helper methods to be used by all tests here...
   VCR.configure do |config|
-    config.allow_http_connections_when_no_cassette = false
+    config.allow_http_connections_when_no_cassette = true
```

Comment:
Nogen grund til at den skal være true?

Reply:
Ellers skal der bruges VCR på alle system test der kører lokalt.

Reply:
Det er faktisk også ideen. Alle tests der kalder eksterne systemer skal have en VCR, for at sikre at tests ikke er sløve eller fejler på grund af en extern service.